executor/explainfor_test.go

```diff
@@ -112,7 +112,7 @@ func (s *testSerialSuite) TestExplainFor(c *C) {
 		}
 		c.Assert(buf.String(), Matches, ""+
 			"TableReader_5 10000.00 0 root  time:.*, loops:1, cop_task: {num:.*, max:.*, proc_keys: 0, rpc_num: 1, rpc_time:.*} data:TableFullScan_4 N/A N/A\n"+
-			"└─TableFullScan_4 10000.00 0 cop.* table:t1 tikv_task:{time:.*, loops:0}, scan_detail:.* keep order:false, stats:pseudo N/A N/A")
+			"└─TableFullScan_4 10000.00 0 cop.* table:t1 tikv_task:{time:.*, loops:0, concurrency:1}, scan_detail:.* keep order:false, stats:pseudo N/A N/A")
```

Comment:
tikv_task?
should tikv tasks show concurrency?

Reply:
By default, every TiKV task has concurrency 1, I think it is at least harmless to show it for TiKV tasks, user can just ignore it.

Reply:
you should ask the related PM.

Reply:
I think keeping tikv's result still is better. No changes is better than harmless changes.

Reply:
For TiKV, I will print `threads:N/A` instead